static/js/map.js

```diff
@@ -840,6 +842,11 @@ function customizePokemonMarker(marker, item, skipNotification) {
                 marker.setAnimation(google.maps.Animation.BOUNCE)
             }
         }
+        if (Store.get('remember_show_ivzero') && perfection < 1) {
```

Comment:
Would perfection < 1 be good enough? What if the IV is 0.5% (not sure if this is possible).